src/duckling/version.ts

```diff
@@ -0,0 +1,58 @@
+export const VERSION = "0.0.1";
+export const MAP_VERSION = "1.1";
```

Comment:
I would implement a completely different API. You should write a compareVersions(actualVersion, expectedVersion). It should return a Compatibile, MinorIncompatible, MajorIncompatible enum.